src/components/PageHero/PageHero.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from "react";
+import { PageHeroLayout } from "./PageHeroLayout";
+
+type PageHeroProps = {
+  title: string;
+  description: string;
+  selectedVideoId: string | null;
+  hasVideoError: boolean;
+};
+
+export const PageHero = ({
+  title,
+  description,
+  selectedVideoId,
+  hasVideoError,
+}: PageHeroProps) => {
+  const handleClickHomepage = React.useCallback(
+    (e: React.MouseEvent<HTMLAnchorElement, MouseEvent>) => {
+      // analytics
+      e.preventDefault();
+      window.location.assign("/");
+    },
+    []
+  );
+
+  const handleClickContactSupport = React.useCallback(
+    (e: React.MouseEvent<HTMLAnchorElement, MouseEvent>) => {
+      // analytics
+      e.preventDefault();
+      window.location.assign("/support");
+    },
+    []
+  );
+
+  return (
+    <PageHeroLayout
+      titleNode={title}
+      descriptionNode={description}
+      sideNode={
+        selectedVideoId && (
+          <div className="video-container">
+            {hasVideoError ? (
```

Comment:
In another iteration, I would explore using ErrorBoundary on this level and throw simulated error in the sideNode component if the debug is true.